enterprise_access/apps/api_client/braze_client.py

```diff
@@ -76,3 +76,15 @@ def create_recipient(
             'send_to_existing_only': False,
             'trigger_properties': trigger_properties or {},
         }
+
+    def create_recipient_no_external_id(self, user_email):
+        """
+        Create a Braze recipient dict identified only by an alias based on their email.
+        """
+        return {
+            'attributes': {'email': user_email},
+            'user_alias': {
+                'alias_label': ENTERPRISE_BRAZE_ALIAS_LABEL,
+                'alias_name': user_email,
+            },
+        }
```

Comment:
We have to use this braze recipient structure to send campaign messages for anonymous users - that is, users for whom we don't yet have an `lms_user_id` (which maps to the "external id" in braze).